torch/csrc/inductor/aoti_torch/c/shim.h

```diff
@@ -735,4 +735,11 @@ DEFINE_DTYPE_SPECIALIZATION(bool, bool)
 
 #endif
 
+AOTI_TORCH_EXPORT AOTITorchError aoti_torch_cpu__weight_int4pack_mm_cpu_tensor(
```

Comment:
Can you add the op to https://github.com/pytorch/pytorch/blob/main/torchgen/aoti/fallback_ops.py instead of manually adding one here? If a fallback op is an aten op, adding it to fallback_ops.py is the recommended way.

Reply:
Thanks for the suggestion. Let me do that in the next PR https://github.com/pytorch/pytorch/pull/146756